### ISSUE TITLE
SIBYLL-2.1 event has PID array which contains zeros

### ISSUE THREAD

**Issue by HDembinski** [impy-project/impy#57]
Here is a random event that I just generated. You can see that quite a few particles in the list have pid == 0, which is invalid.
```
SibyllEvent(kin=Event kinematics:
	ecm      :  100.00000
	pcm      :   49.98239
	elab     : 5328.01821
	plab     : 5328.01813
	gamma_cm :   53.28956
	bgamm_cm :   53.28018
	pdgid 1  :       2212
	nucprop 1: 1/1.0
	pdgid 2  :       2212
	nucprop 2: 1/1.0
, frame='center-of-mass', nevent=0, pid=array([   0,    0,  211, 2212,    0,  321,    0,    0,  211, 2112, -211,
          0,  211, -211, 3122,  211, -211,    0,  211, -211,    0,   22,
         22,   22,   22,   22,   22], dtype=int32), status=array([2, 2, 1, 1, 2, 1, 2, 2, 1, 1, 1, 2, 1, 1, 1, 1, 1, 2, 1, 1, 2, 1,
       1, 1, 1, 1, 1], dtype=int32), charge=array([ 0,  0,  1,  1,  0,  1,  0,  0,  1,  0, -1,  0,  1, -1,  0,  1, -1,
        0,  1, -1,  0,  0,  0,  0,  0,  0,  0], dtype=int32), px=array([-0.63597113,  0.30926612, -0.57559907,  0.46384686,  0.51563632,
        0.14977658, -0.34058014,  0.03916309, -0.233496  ,  0.30795735,
       -0.05713884, -0.57883763,  0.48268995, -0.17342119,  0.47702566,
        0.03861185, -0.45473272,  0.11414974,  0.23528023, -0.06781755,
       -0.12830749, -0.30247974, -0.27651668,  0.13449517, -0.02031411,
       -0.13858746,  0.01024476]), py=array([-0.06726643,  0.21742968, -0.43126103,  0.17648369, -0.26430294,
        0.72727096, -0.14105777, -0.0387254 , -0.28010216,  0.10153134,
        0.15356468, -0.22083166,  0.23095961, -0.01352806, -0.03491727,
       -0.22938628,  0.02613074, -0.1671897 , -0.08275045,  0.02067954,
        0.02334706, -0.18664402, -0.03424823, -0.0877149 , -0.07952067,
        0.05718039, -0.03382691]), pz=array([ 1.59013569e+00,  1.86574388e+00,  8.69777298e+00,  2.52620697e+01,
        5.80550623e+00, -2.51288414e-02,  2.31336451e+00,  3.51808786e+00,
       -3.89408565e+00, -4.51334534e+01,  6.23095334e-02,  1.52783954e+00,
        1.65816951e+00,  2.07590282e-01,  4.39044857e+00,  1.41507137e+00,
        2.05674314e+00,  2.56640792e-01,  2.39994287e+00,  2.20560968e-01,
        8.97658288e-01,  8.76120746e-01,  6.52137876e-01,  1.56916633e-01,
        9.97945592e-02,  7.78088510e-01,  1.19816080e-01]), en=array([ 1.87824392,  2.05776715,  8.72942257, 25.28435707,  5.99597931,
        0.91774952,  2.47411704,  3.60436106,  3.91361141, 45.14439392,
        0.22407411,  1.65418553,  1.75117266,  0.30661198,  4.55515432,
        1.44083834,  2.11730909,  0.35682869,  2.41690087,  0.27046952,
        0.91706872,  0.94547206,  0.7091673 ,  0.22630349,  0.13062309,
        0.79239994,  0.12492038]), m=array([0.7683    , 0.76859999, 0.13957   , 0.93826997, 1.38279998,
       0.49364999, 0.7683    , 0.78195   , 0.13957   , 0.93957001,
       0.13957   , 0.13496999, 0.13957   , 0.13957   , 1.11563003,
       0.13957   , 0.13957   , 0.13496999, 0.13957   , 0.13957   ,
       0.13496999, 0.        , 0.        , 0.        , 0.        ,
       0.        , 0.        ]), vx=array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0.]), vy=array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0.]), vz=array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0.]), vt=array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0., 0., 0., 0., 0., 0.]), parents=None, children=None)
```

**Comment by HDembinski** [comment]
The good news: the invalid particles have status == 2, so are not final state. Perhaps we can leave it like this if this not a bug in our conversion of SIBYLL's stack to HEPEVT.

**Comment by afedynitch** [comment]
Since I restored the PIDs for decayed particles in the fortran layer, this should be closed by #54 .

**Comment by HDembinski** [comment]
fixed in #54
